### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by DiamondNath07**
### Essence of this PR
This PR displays the graphical user interface of a calculator application taking into consideration the following;

- [ ] Exact wireframe for the project requirement
- [ ] No additional styling
- [ ] Used js to style app from the calculator.js file

Thanks for reviewing👏

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @DiamondNath07, KIndly add the future features section of your app to your readme, all headers from the microverse template should be present in your readme and the content personalized

**Comment by prolajumokeoni** [comment]
Hi @DiamondNath07 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the future features section in your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```
Comment (prolajumokeoni) [inline src/App.js]: - [ ] Kindly make the App.js component class-based this is a requirement for this milestone 👍🏿
- Currently, this file is deleted please restore it back and ensure  `App.js`(main component)  has the `Calculator.js` component  imported 👍🏿

File: src/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            >
+              {btn.val}
+            </button>
+          ))}
+        </div>
+      </div>
+    </>
+  );
+};
+
```
Comment (prolajumokeoni) [inline src/Calculator.js]: - [ ] Kindly make the Calculator.js component class-based this is a requirement for this milestone 👍🏿
- [ ] Please create a directory called `components` and Calculator.js should be inside this directory  👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @DiamondNath07 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Please delete the boilerplate that got created by the CRA (texts, images, and styles), please remove images and styles not used in your app
- [ ] Based on the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#project-requirements), create a directory called components and your Calculator.js should be inside this directory


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @DiamondNath07 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE





 
 ## Optional suggestions
 
 - Please check the comment under the review.
 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread  and tag me algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**